src/controllers/eventController.js

```diff
@@ -59,6 +65,10 @@ eventController.get('/', async (req, res) => {
 eventController.get('/:id', async (req, res) => {
     try {
         const event = await findEventByID(req.params.id);
+        
+        if (event?.isApproved === false && req.requester?._id != event.creator._id) {
```

Comment:
Once you are using loose equality and right after it - strict equality? But what logic are you trying to achieve with this error?

Reply:
This is to check, IF there are requester and he is creator to see about his Event, in cases that you want to edit it, does not matter if is approved or not.

Reply:
This is the main reason, this is in case if event does not exist.
![image](https://github.com/Getsov/Motorsport-Events-Server/assets/50380455/de2715f3-c596-488d-ad95-93855564e3d7)

Reply:
Okay, but still - don't use, especially at once, the two types of check - loose and strict.

Reply:
fixed - if event is not approved and someon which is not owner try to view it , the server will return error:
         throw new Error('This Event is not Approved by Admin!');
        Server will return not approved event only if the requester is the owner